libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.csproj

```diff
@@ -37,6 +37,10 @@
     <Content Include="**/*.qna" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Schemas\Microsoft.Test.MemoryAssertions.schema" />
+  </ItemGroup>
```

Comment:
This will go away when you merge with main.

Reply:
If you have already merge with main, you should remove this.

---
In reply to: [491100013](https://github.com/microsoft/botbuilder-dotnet/pull/4640#discussion_r491100013) [](ancestors = 491100013)